plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/GcsStorageFactory.java

```diff
@@ -70,10 +70,10 @@ public GcsStorageFactory(HdfsEnvironment hdfsEnvironment, HiveGcsConfig hiveGcsC
         }
     }
 
-    public Storage create(ConnectorSession session, Path path)
+    public Storage create(ConnectorSession session, String path)
     {
         try {
-            GoogleCloudStorageOptions gcsOptions = TrinoGoogleHadoopFileSystemConfiguration.getGcsOptionsBuilder(hdfsEnvironment.getConfiguration(new HdfsContext(session), path)).build();
+            GoogleCloudStorageOptions gcsOptions = TrinoGoogleHadoopFileSystemConfiguration.getGcsOptionsBuilder(hdfsEnvironment.getConfiguration(new HdfsContext(session), Locations.getUri(path))).build();
```

Comment:
We probably need to follow-up on the usage of `hdfsEnvironment.getConfiguration(...)` method.

Is there any plan/idea how to cope with this dependency?

cc @electrum